src/utils/selectors.ts

```diff
@@ -40,32 +40,21 @@ export const profileSelector: SparkleSelector<FirebaseReducer.Profile<User>> = (
  *
  * @param state the Redux store
  */
-export const currentVenueSelector: SparkleSelector<WithId<AnyVenue>> = (
-  state
-) => state.firestore.ordered.currentVenue?.[0];
+export const currentVenueSelector: SparkleSelector<
+  WithId<AnyVenue> | undefined
+> = (state) => state.firestore.ordered.currentVenue?.[0];
 
 // @debt can we merge this with currentVenueSelector and just use 1 canonical version?
 export const currentVenueSelectorData: SparkleSelector<AnyVenue | undefined> = (
   state
 ) => state.firestore.data.currentVenue;
 
-/**
- * Selector to retrieve partygoers from the Redux Firestore.
- *
- * @param state the Redux store
- */
-export const partygoersSelector: SparkleSelector<WithId<User>[] | undefined> = (
-  state
-) => state.firestore.ordered.partygoers;
+export const usersSelector: SparkleSelector<WithId<User>[]> = (state) =>
+  state.firestore.ordered.users ?? [];
 
-/**
- * Selector to retrieve partygoers from the Redux Firestore.
- *
- * @param state the Redux store
- */
-export const partygoersSelectorData: SparkleSelector<
-  Record<string, User> | undefined
-> = (state) => state.firestore.data.partygoers;
```

Comment:
Why was this removed? And if it's being replaced with `usersByIdSelector ` below, would be nice not to lose the documentation comments.

Reply:
They were not used, `hooks grouping discussion`

Reply:
If you think `Selector to retrieve partygoers from the Redux Firestore.` can add value, I can leave the comments

Reply:
fixed in e7c05d2e2c040fd58dcda1e648598489d528fd44